scripts/aggregate-json-files.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+instances=(brb dbc nbc)
+environments=(dev ref)
```

Comment:
Where is environments used in this script?

Reply:
fixed